java-codegen/opensearch-openapi.yaml

```diff
@@ -43665,6 +43665,8 @@ components:
                 - wildcard
             null_value:
               type: string
+            normalizer:
+              type: string
```

Comment:
This file is produced and retrieved from the opensearch-api-specification repository, so this change also needs to be made here: https://github.com/opensearch-project/opensearch-api-specification/blob/main/spec/schemas/_common.mapping.yaml#L499
Otherwise it will be overwritten

Reply:
https://github.com/opensearch-project/opensearch-api-specification/pull/847